incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for redis-cache.
```

Comment:
I'd suggest you structure it by component. A good example for this is https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/values.yaml. This also shows you how to configure resources in a values file.